cmd/main.go

```diff
@@ -8,7 +9,7 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 	logging "github.com/sirupsen/logrus"
-	"github.com/thought-machine/aws-service-quotas-exporter/pkg/service_exporter/serviceexporter"
+	"github.com/thought-machine/aws-service-quotas-exporter/serviceexporter"
```

Comment:
nit: since `serviceexporter` module is clearly under service-quotas-exporter why don't we rename it to `github.com/thought-machine/aws-service-quotas-exporter/exporter` ?

Reply:
Good idea! Will do that in another PR :)